src/option_learning.py

```diff
@@ -212,3 +220,206 @@ def update_segment_from_option_spec(self, segment: Segment,
                                   dtype=np.float32)
                 option = param_opt.ground([robby, block], params)
                 segment.set_option(option)
+
+
+@dataclass(frozen=True, eq=False, repr=False)
+class _LearnedSimpleParameterizedOption(ParameterizedOption):
+    """A convenience class for holding a learned parameterized option.
```

Comment:
I would wipe this entire docstring and replace it with a description of the algorithm behind this "simple" approach! It's def not a convenience class lol